src/IceRpc/Transports/Slic/Internal/SlicConnection.cs

```diff
@@ -1190,8 +1205,8 @@ async Task ReadPongFrameAsync(int size, CancellationToken cancellationToken)
                     (ref SliceDecoder decoder) => new PongBody(ref decoder),
                     cancellationToken).ConfigureAwait(false);
 
-                // For now, we only send a 0 payload value.
-                if (pongBody.Payload != 0L)
+                // For now, we only send a 0 and 1 payload value.
```

Comment:
```suggestion
                // For now, we only send a 0 or 1 payload value.
```

Reply:
We should document somewhere that 0L is for write timer pings, and 0 is for read timer. Is there a benefit to using a different value?